memberships/templates/memberships/register.html

```diff
@@ -2,6 +2,21 @@
 
 {% block content %}
   <h2>Member Registration</h2>
+  <script>
+/*Execute recaptchaV3 function*/
+grecaptcha.ready(function() {
+  grecaptcha.execute('6LfGd9oZAAAAAEx2_gzbS1cR8X0_IEmPNFG8A9eN', {action: "/register/"}).then(function(token) {
```

Comment:
Could we pull these keys into environment config and then only add this code to the page if those keys are available? That gives us two things:
1. During deployment we can securely add the API keys to the machine without them being committed to the git repoi
2. Developer machines will only have recaptcha if they want them to. That is especially helpful if you are running automated tests that will probably be rejected by recaptcha

There are some downsides to this of course, mainly:
1. Developers are less likely to implicitly test recaptcha when doing development as it is disabled
2. There will have to be extra code in the view to account for the recaptcha key not being sent

Reply:
Another good reason to do this is that this causes the tests to fail as they do not submit a valid recaptcha :)

I suppose we could mock out google, but I tend to prefer avoiding mocks where possible.

Reply:
Thanks! Will get working on it